src/sentry/rules/actions/notify_event_sentry_app.py

```diff
@@ -29,7 +29,7 @@ def validate_field(value: str | None, field: Mapping[str, Any], app_name: str) -
             )
 
 
-class NotifyEventSentryAppAction(EventAction):
+class NotifyEventSentryAppAction(SentryAppEventAction):
```

Comment:
This is just to clean up the code, since a lot of places in the  code check if `id in SENTRY_APP_ACTIONS` then assume a bunch of methods to be present because we know currently there's only one rule in `SENTRY_APP_ACTIONS`.

This way, if we add to `SENTRY_APP_ACTIONS` we can just extend this same `SentryAppEventAction` class and already know what methods we need to extend.